modulemd/modulemd-util.c

```diff
@@ -360,7 +360,7 @@ modulemd_validate_nevra (const gchar *nevra)
         {
           break;
         }
-      else if (*i == '-')
+      if (*i == '-')
```

Comment:
Why this?

Reply:
The code formatter that we run during `ninja test` recently started to drop unneeded `else` statements. It's unrelated to this change. I've pushed this to the current master branch so it doesn't keep showing up.